docs/Reference/Plugin-API-Interfaces.md

```diff
@@ -0,0 +1,36 @@
+description: Plugin interfaces
+<!--- END of page meta data -->
+
+# Plugin API Interfaces
+
+Hyperledger Besu provides API interfaces to allow users to [build plugins](../Concepts/Plugins.md) to
```

Comment:
Hyperledger Besu provides API interfaces to allow users -> API interfaces in Hyperledger Besu allow users 

Suggested edit to make more active

Reply:
Done